<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="rightPanel" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>401</width>
      <height>431</height>
     </rect>
    </property>
    <widget class="QPushButton" name="openButton">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>338</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>368</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Info</string>
     </property>
    </widget>
    <widget class="QLabel" name="DescLabel">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>174</y>
       <width>361</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>No files are opened.</string>
     </property>
     <property name="buddy">
      <cstring>DescLabel</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="ImageLabel">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>361</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>ImageLabel</string>
     </property>
    </widget>
    <widget class="QComboBox" name="moreActions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Select Actions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Delete File</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Clone File</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="currentRootDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QTreeView" name="filelistView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>341</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Directory"/>
    <addaction name="actionOpen_Directory"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionabout"/>
    <addaction name="actionHow_to_use"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuSort_By">
     <property name="title">
      <string>Sort By</string>
     </property>
     <addaction name="actionType"/>
     <addaction name="actionSize"/>
     <addaction name="actionDate_Created"/>
    </widget>
    <widget class="QMenu" name="menuFonts">
     <property name="title">
      <string>Fonts</string>
     </property>
     <widget class="QMenu" name="menuChange_Font_Style">
      <property name="title">
       <string>Change Font Style</string>
      </property>
      <addaction name="actionDefault_Font"/>
      <addaction name="actionHeebo"/>
      <addaction name="actionArial"/>
      <addaction name="actionTimes_New_Roman"/>
      <addaction name="actionConsolas"/>
     </widget>
     <addaction name="menuChange_Font_Style"/>
     <addaction name="actionIncrease_Font_Size"/>
     <addaction name="actionDecrease_Font_Size"/>
    </widget>
    <addaction name="menuSort_By"/>
    <addaction name="menuFonts"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuPassword_Protection">
     <property name="title">
      <string>Password Protection</string>
     </property>
     <addaction name="actionSet_Password"/>
     <addaction name="actionChange_Password"/>
    </widget>
    <addaction name="menuPassword_Protection"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_File">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionNew_Directory">
   <property name="text">
    <string>New Directory</string>
   </property>
  </action>
  <action name="actionOpen_Directory">
   <property name="text">
    <string>Open Directory</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionThemes">
   <property name="text">
    <string>Themes</string>
   </property>
  </action>
  <action name="actionHow_to_use">
   <property name="text">
    <string>How to use?</string>
   </property>
  </action>
  <action name="actionType">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </action>
  <action name="actionSize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Size</string>
   </property>
  </action>
  <action name="actionDate_Created">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Date Created</string>
   </property>
  </action>
  <action name="actionSet_Password">
   <property name="text">
    <string>Set Password</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="text">
    <string>Change Password</string>
   </property>
  </action>
  <action name="actionFile_Deletion_Confirmation">
   <property name="text">
    <string>File Deletion Confirmation</string>
   </property>
  </action>
  <action name="actionDefault_File_Format">
   <property name="text">
    <string>Default File Format</string>
   </property>
  </action>
  <action name="actionDefault_Folder_Location">
   <property name="text">
    <string>Default Folder Location</string>
   </property>
  </action>
  <action name="actionReset_Settings">
   <property name="text">
    <string>Reset Settings</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionIncrease_Font_Size">
   <property name="text">
    <string>Increase Font Size</string>
   </property>
  </action>
  <action name="actionDecrease_Font_Size">
   <property name="text">
    <string>Decrease Font Size</string>
   </property>
  </action>
  <action name="actionDefault_Font">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string> Default Font- Segoe UI</string>
   </property>
  </action>
  <action name="actionHeebo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Heebo</string>
   </property>
   <property name="font">
    <font>
     <family>Heebo</family>
    </font>
   </property>
  </action>
  <action name="actionArial">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Arial</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionTimes_New_Roman">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Times New Roman</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
    </font>
   </property>
  </action>
  <action name="actionConsolas">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Consolas</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
